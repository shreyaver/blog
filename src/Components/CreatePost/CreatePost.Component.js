import React from 'react';
import './CreatePost.Component.css';
import Header from '../Header/Header.Component.js';
import Footer from '../Footer/Footer.Component';
import PostForm from '../PostForm/PostForm.Component.js';

const CreatePost = (props) => {
  return (
    <div className = "CreatePost">
      <Header />
      <PostForm />
      <Footer />
    </div>
  );
}

export default CreatePost;
