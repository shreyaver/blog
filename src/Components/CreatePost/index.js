import React from 'react';
import './CreatePost.css';
import Header from '../Header';
import Button from '../Button';
import Footer from '../Footer';
import PostForm from '../PostForm';

const CreatePost = (props) => {
  return (
    <div className = "CreatePost">
      <Header />
      <PostForm />
      <Footer />
    </div>
  );
}

export default CreatePost;
