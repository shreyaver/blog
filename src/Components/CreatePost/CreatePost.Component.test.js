import React from 'react';
import renderer from 'react-test-renderer';
import CreatePost from './CreatePost.Component';

describe('CreatePost', () => {
  it('renders without crashing', () => {
    const createPost = renderer.create(<CreatePost />).toJSON();
    expect(createPost).toMatchSnapshot();
  });
})
