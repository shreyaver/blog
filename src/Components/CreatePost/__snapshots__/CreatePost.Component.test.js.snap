// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePost renders without crashing 1`] = `
<div
  className="CreatePost"
>
  <header>
    <div
      className="Content"
    >
      <div>
        <h1>
          The Artifact
        </h1>
        <h2>
          Culture & Art blog
        </h2>
      </div>
      <nav>
        <div
          className="navBar"
        >
          <a
            href="/"
          >
            Blog
          </a>
          <a
            href="#"
          >
            About
          </a>
          <a
            href="#"
          >
            Contact
          </a>
        </div>
      </nav>
    </div>
  </header>
  <form
    className="Form-content"
    encType="multipart/form-data"
    onSubmit={[Function]}
  >
    <span>
      <label>
        Date
      </label>
      <input
        className="date"
        onChange={[Function]}
        type="date"
        value=""
      />
    </span>
    <span>
      <label>
        Reading time (in minutes)
      </label>
      <input
        className="readingTime"
        onChange={[Function]}
        type="number"
      />
    </span>
    <span
      className="titleEntry"
    >
      <label>
        Title (Characters: 
        70
        )
      </label>
      <input
        className="title"
        maxLength={70}
        onChange={[Function]}
        type="text"
        value=""
      />
    </span>
    <span
      className="descriptionEntry"
    >
      <label>
        Description (Characters: 
        130
        )
      </label>
      <textarea
        className="description"
        maxLength={130}
        onChange={[Function]}
        value=""
      />
    </span>
    <span>
      <label>
        Image
      </label>
      <input
        accept="image/*"
        name="image"
        type="file"
      />
    </span>
    <span>
      <button
        className="Button"
        type="submit"
      >
        UPLOAD POST
      </button>
    </span>
  </form>
  <footer>
    <div
      className="footerContent"
    >
      <small>
        Â©artifact.com 2019
      </small>
    </div>
  </footer>
</div>
`;
